<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::TagProcessingTargetsOptionPage</class>
 <widget class="QWidget" name="QtGui::TagProcessingTargetsOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Targets</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="toBeAddedLabel">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Target levels to be added when opening a file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="targetsToBeAddedListView">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
Remarks
&lt;ul style=&quot;margin-left: -20px; margin-top: -3px; text-indent: 0px &quot;&gt;
&lt;li&gt;It is ensured that at least one tag for each selected target level is present when opening a file.&lt;/li&gt;
&lt;li&gt;Has no effect if automatic tag management is disabled.&lt;/li&gt;
&lt;li&gt;Has no effect when targets are not supported by the file format.&lt;/li&gt;
&lt;ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
